<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsWindow</class>
 <widget class="QDialog" name="SerialSettingsWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Serial Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="serialSettingsForm">
     <item row="0" column="0">
      <widget class="QLabel" name="baudrateLabel">
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="baudrateCbox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="databitsLabel">
       <property name="text">
        <string>Databits</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="flowctrlLabel">
       <property name="text">
        <string>Flow Control</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="parityLabel">
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stopbitsLabel">
       <property name="text">
        <string>Stop Bits</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="databitsCbox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="flowctrlCbox"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="parityCbox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="stopbitsCbox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="serialSettingsButtonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
